user/pages/04.Reference/10.get-quota-info/docs.en.md

```diff
@@ -216,12 +235,15 @@ The response contains the information about the currently-used resources in JSON
     "compute.instances": 9,
     "compute.ram_mb": 204800,
     "dns.zones": 2,
+    "loadbalancer.flavors": {
+      "failover-small": 2
+    },
+    "loadbalancer.healthmonitors": 2,
+    "loadbalancer.listeners": 2,
+    "loadbalancer.loadbalancers": 2,
```

Comment:
should we mock up an example with two different loadbalancer flavors where 1+2=3?

Reply:
done